src/vendor_upstream/dom/translateDOMPositionXY.js

```diff
@@ -29,16 +30,21 @@ var translateDOMPositionXY = (function() {
     // Use 2D translation instead.
     if (!isSafari && BrowserSupportCore.hasCSS3DTransforms()) {
       return function(/*object*/ style, /*number*/ x, /*number*/ y) {
-        style[TRANSFORM] ='translate3d(' + x + 'px,' + y + 'px,0)';
```

Comment:
Why do we need this function to be Locale specific?  I would hope it could remain general to both cases.

Reply:
My basic strategy with this PR was to reverse DOM events as necessary, and reverse render positions/translations as necessary. This way, it'd keep the app state still the same (e.g. moving a column to the right in LTR is positive dragDistance, as would be moving a column to the left in RTL).

Reply:
Would it be possible to wrap the call to this method and make this change there?  Ideally these vendor_upstream files should be replaced by fbjs where they are now maintained, so I'd prefer not to customize them too much.